index.html

```diff
@@ -1,192 +1,480 @@
+        <li>
+          <a
+            class="m"
+            href="https://medium.com/"
+            rel="noopener noreferrer"
+            target="_blank"
+            ><img alt="logo-m" width="22px" height="20px" src="images/m.png"
+          /></a>
+        </li>
+      </ul>
+      <button type="button" class="my-resume">Get my resume</button>
+      <ul>
+        <li>
+          <div class="languages">
+            <p class="lang poppins">Languages</p>
+            <button name="button-arrow1" type="button">
+              <img
+                class="arrow-down"
+                alt=""
+                width="512px"
+                height="512px"
+                src="images/down-arrow.png"
+              />
+            </button>
+          </div>
+          <ul class="lang-sub">
+            <!-- <div class="lang1"> -->
+            <li class="li-sp">
+              <div class="row1">
+                <img
+                  class="js-img"
+                  alt="JS Logo"
+                  width="100px"
+                  height="100px"
+                  src="images/js.png"
+                />
+                <p class="js poppins">JavaScript</p>
+              </div>
+            </li>
+            <li class="li-sp">
+              <div class="row1">
+                <img
+                  class="html5-img"
+                  alt="HTML5 Logo"
+                  width="100px"
+                  height="101px"
+                  src="images/html5.png"
+                />
+                <p class="html5 poppins">HTML</p>
+              </div>
+            </li>
+            <li class="li-sp">
+              <div class="row1">
+                <img
+                  class="css3-img"
+                  alt="CSS3 Logo"
+                  width="100px"
+                  height="101px"
+                  src="images/css3.png"
+                />
+                <p class="css3 poppins">CSS</p>
+              </div>
+            </li>
+            <!-- </div> -->
+          </ul>
+          <hr />
+        </li>
+        <li>
+          <div class="frameworks">
+            <p class="frame poppins">Frameworks</p>
+            <button name="button-arrow2" type="button">
+              <img
+                class="arrow-right"
+                alt=""
+                width="512px"
+                height="512px"
+                src="images/right-arrow.png"
+              />
+            </button>
+          </div>
+          <hr />
+        </li>
+        <li>
+          <div class="skills">
+            <p class="skill poppins">Skills</p>
+            <button name="button-arrow3" type="button">
+              <img
+                class="arrow-right"
+                alt=""
+                width="512px"
+                height="512px"
+                src="images/right-arrow.png"
+              />
+            </button>
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
```

Comment:
While you made great effort on this block, but there are a few things you have to fix before going to the next steps.

- Kindly, I suggest you remove the `section` tag inside the `footer` tag for good semantics and best practices. 😄